util/strings.go

```diff
@@ -37,3 +37,25 @@ func IndexWhiteSpace(s string) int {
 	}
 	return -1
 }
+
+// see https://confluence.atlassian.com/jiracoreserver073/search-syntax-for-text-fields-861257223.html
+func EscapeJiraReservedChars(in string) string {
```

Comment:
Can you change the name of this and the test to reflect that this escaping is for search queries/JQL?